org.uqbar.project.wollok/src/org/uqbar/project/wollok/validation/WollokDslValidator.xtend

```diff
@@ -978,6 +979,14 @@ class WollokDslValidator extends AbstractConfigurableDslValidator {
 			report(message, it, WNAMED__NAME)
 		]
 	}
+	
+	@Check
+	@DefaultSeverity(ERROR)
```

Comment:
¿Está bien poner error? ¿No sería warning en función de los últimos cambios que hablamos?

Reply:
Sí, que sea Warning @Guusy... `(WARN)` en lugar de `(ERROR)`

Reply:
Buenas justo con este tema me pasa lo siguiente en cuanto a los test, le cambie a warning dentro del test y lo hice correr justamente para que fallen, pero no estarian fallando, adjunto gif, disculpen la mala calidad del gif, si alguien me recomienda alguna app para ubuntu de gifs, joya, estoy usando peeks.
Igualmente capaz xpect solo chequea el mensaje y no el tipo si es `error` o `warning`, la verdad no conozco realmente.

![wollok_test_bug](https://user-images.githubusercontent.com/23525653/61595861-c51ced80-abd2-11e9-8662-2cf609564613.gif)

Reply:
Recientemente vi que me pasó eso, incluso si le ponés un espacio: `Ob ject is a reserved ...` tampoco salta el error, muy loco que le aplica un trim como algoritmo. Por eso no te preocupes, ponele `WARN` en lugar de `ERROR` y listo. 

Respecto a grabar con gifs, para mí Peek es la opción más liviana y la que estoy usando, antes usaba silentcast pero si el gif es muy grande te mata la compu. Otra opción piola es achicar la pantalla para que el gif no te quede muy grande (o seleccionar una parte en Peek para que no grabe toda la pantalla)

Reply:
Muchas gracias, voy por peek :rocket: